yarn-project/ivc-integration/serve.mt.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
FYI the browser test does not need a running server to execute. But I do add a a webpage as in browser test app to serve the app for execution in a browser bc I want to be able to use the  browser's built in profiling and debugging tools.